actions.py

```diff
@@ -184,6 +185,12 @@ def calculate_emissions(self, tracker, unit="short_tons"):
 
         departure_airport = self.find_airport_by_ref(departure_iata)
         destination_airport = self.find_airport_by_ref(destination_iata)
+        if stopover_iata:
```

Comment:
Optional: Maybe we could use a list `travel_route` that stores departure, destination, and stopover together instead? This would reduce the amount of code and also make it relatively easy to generalise to multiple stopovers.